src/routes/mainRouter.js

```diff
@@ -10,4 +17,12 @@ router.post("/signup", signupDataValidate, signUp);
 router.route("/login").post(loginDataValidate, logIn).get(authenticateUser);
 router.get("/logout", logOut);
 
+//notes
+
+router.post("/addnote", authenticateUser, addNote);
+router.get("/getnote", authenticateUser, getNotes);
```

Comment:
I think it can be confusing that here it says `/getnote` but it actually can give multiple notes of one user

Reply:
Got it , Sir !